ext/standard/base64.c

```diff
@@ -1229,12 +1233,15 @@ PHP_FUNCTION(base64_encode)
 	char *str;
 	size_t str_len;
 	zend_string *result;
+	bool padding = true;
 
-	ZEND_PARSE_PARAMETERS_START(1, 1)
+	ZEND_PARSE_PARAMETERS_START(1, 2)
 		Z_PARAM_STRING(str, str_len)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_BOOL(padding)
 	ZEND_PARSE_PARAMETERS_END();
 
-	result = php_base64_encode((unsigned char*)str, str_len);
+	result = php_base64_encode_ex((unsigned char*)str, str_len, (padding ? 0 : PHP_BASE64_NO_PADDING));
```

Comment:
I guess you used a flag for future proofing in case a new option would be needed in the future, right? Personally, I'd be fine with a bool value, but I can also understand your POV :)

Reply:
Yes (see discussion about URL SAFE mode) and to avoid changing function prototype again in so much places ;)